open-banking-brasil-financial-api-1_ID1.md

```diff
@@ -211,8 +211,8 @@ In addition, the Authorization Server
 1. shall support a signed and encrypted JWE request object passed by value or shall require pushed authorization requests [PAR];
 2. shall distribute discovery metadata (such as the authorization endpoint) via the metadata document as specified in [OIDD] and [RFC8414]
 3. shall support the claims parameter as defined in clause 5.5 [OpenID Connect Core][OIDC]
-4. shall support the oidc standard claim "cpf" as defined in clause 5.2.2.2 of this document
-5. shall support the oidc standard claim "cnpj" as defined in clause 5.2.2.3 of this document
+4. shall support the oidc standard claim "cpf" as defined in clause 5.2.2.2 of this document if providing access to resources where the resource owner is a `natural person`
+5. shall support the oidc standard claim "cnpj" as defined in clause 5.2.2.3 of this document if providing access to resources where the resource owner is a `legal person` but not a `natural person`
```

Comment:
@RalphBragg could you please check if the text is ok in English? We (GT Sec) think the item 4 doesn't need any change.

5. shall support the oidc standard claim "cnpj" as defined in clause 5.2.2.3 of this document if providing access to resources where the resource owner is an account holder (ASPSP) for business entities.

Reply:
Great! 
Could you review the PR?